<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>expence</class>
 <widget class="QMainWindow" name="expence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#d3ffc5;   /* Mint green */</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Food</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Entertainment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Traveling</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clothing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Education Equipment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>University fees</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>392</width>
      <height>236</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AllExpence">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>630</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>See all expence details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 28px;            /* Larger font size for emphasis */
    color: #4CAF50;             /* Green color for a fresh look */
    font-weight: bold;          /* Make the text bold */
    padding: 10px 20px;         /* Add padding around the text */
    background-color: #e8f5e9;  /* Light green background to match the color scheme */
    border-radius: 8px;         /* Rounded corners */
    text-align: center;         /* Center-align the text */
}
</string>
    </property>
    <property name="text">
     <string>Expence Tracker</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
